js/views/composer.js

```diff
@@ -187,6 +208,22 @@ define(function(require) {
 				_this.saveDraft();
 			}, this.draftInterval);
 		},
+		toggleEditor: function()
```

Comment:
trumbowyg only has viewhtml functionality. but we can add a plugin for that  if we want, in that case, any example links on how it should look like